No hint path defined for [google2fa]

After following your installations steps, I get the following error message.

```
No hint path defined for [google2fa]. (View: resources/views/vendor/spark/settings/security/enable-two-factor-auth.blade.php) (View: resources/views/vendor/spark/settings/security/enable-two-factor-auth.blade.php) (View: resources/views/vendor/spark/settings/security/enable-two-factor-auth.blade.php)
```

The only clue I can find, is that the last command that I should've run, ```npm run dev```,  failed on me;

```
$ npm run dev
npm ERR! Darwin 16.4.0
npm ERR! argv "/usr/local/Cellar/node/7.5.0/bin/node" "/usr/local/bin/npm" "run" "dev"
npm ERR! node v7.5.0
npm ERR! npm  v4.1.2

npm ERR! missing script: dev
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     npm-debug.log
```

Any clues as to where I could go looking?

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # Google Authenticator support for Laravel Spark
 
-This package replaces the default Two-Factor Authentication driver with [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en). 
+This package replaces the default Two-Factor Authentication driver with [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en).
 
 ![Demo](http://i.imgur.com/WQKx5nS.gif)
 
@@ -9,6 +9,8 @@ This package replaces the default Two-Factor Authentication driver with [Google 
 - Run `composer require eusebiu/laravel-spark-google2fa`
 
 - Add `Eusebiu\LaravelSparkGoogle2FA\Google2FAServiceProvider::class` to your `providers` array in `config/app.php` after `SparkServiceProvider`.
+
+- Run `php artisan config:clear`
 
 - Run `php artisan vendor:publish --provider="Eusebiu\LaravelSparkGoogle2FA\Google2FAServiceProvider" --tag=migrations --tag=assets`
 

